packages/tools/devtools/devtools-view/src/components/TooltipTexts.tsx

```diff
@@ -37,3 +37,23 @@ export const containerStatusTooltipText = (
 		.
 	</div>
 );
+
```

Comment:
Looking for feedback on the tooltip contents here. I don't have a strong understanding of the semantics / implications of these beyond what is in the code comments.

Additionally, if we have other public documentation on these concepts, I am not aware of them. But I would love to link them if we do.

Reply:
Client mode matches my understanding. If anything, I'd add `Note that the current connection mode does not necessarily reflect the client's permissions. A client with write permissions can connect to a container in view-only mode if it wishes.`.

The scopes piece I'm not familiar with, so can't comment there, but the current content feels ok.